src/core/server/saved_objects/migrationsv2/actions/remove_write_block.ts

```diff
@@ -39,7 +39,7 @@ export const removeWriteBlock = ({
         // Don't change any existing settings
         preserve_existing: true,
         body: {
-          index: {
+          settings: {
             blocks: {
```

Comment:
Is this an actual API change?

Reply:
According to the Elasticsearch tests, both forms are valid

https://github.com/elastic/elasticsearch/blob/27089e74602dc442656cf97ca4b29af79f724933/qa/full-cluster-restart/src/test/java/org/elasticsearch/upgrades/FullClusterRestartIT.java#L560
https://github.com/elastic/elasticsearch/blob/27089e74602dc442656cf97ca4b29af79f724933/qa/full-cluster-restart/src/test/java/org/elasticsearch/upgrades/FullClusterRestartIT.java#L301

I picked one supported by `@elastic/elasticsearch` client